index.html

```diff
@@ -44,23 +44,20 @@
         <dl class="info-headings">
           <div>
             <dt>Data Gathered Since</dt>
-              <!-- @todo get date-start from store -->
-            <dd id="date-start">
-              <time datetime="2017-06-13T11:00">June 13, 2017</time>
+            <dd>
+              <time id="data-gathered-since" datetime="2017-06-13T11:00"></time>

```

Comment:
Okay; fixed this thank you. I converted the Date object to an ISO string that looks to be an acceptable format, `2017-08-23T17:21:54.495Z` as an example value for the `datetime` attribute on the `time` element...